src/components/AddressResolver/AddressesTable.tsx

```diff
@@ -11,7 +11,7 @@ import { EndpointEntry } from "./EndpointEntry";
 export const TableStyle = (): string => {
   return `
     .table-responsive {
-      border: solid 1px ${vars.greyLight};
+      border: solid 1px ${vars.grey300};
```

Comment:
actually since we are at it, wanna change this to use the twin marco?
```
${tw`border border-solid border-grey-300`}
```

something like that

Reply:
yeah im thinking to refactor these further in the next pr